### ISSUE TITLE
ANSI control sequence ESC M uses incorrect default background color (Windows Console Host)

### ISSUE THREAD

**Issue by 12Me21** [microsoft/terminal#4182]
# Environment

```none
Microsoft Windows [Version 10.0.18362.535]
```

# Steps to reproduce

Screen background is set to any color other than 0:
![image](https://user-images.githubusercontent.com/4053256/72211525-e03e2b00-349a-11ea-8645-50e1770e13ca.png)

From bash in WSL:
```
clear
echo -e "\e[H\e[1T"
```
![image](https://user-images.githubusercontent.com/4053256/72211510-af5df600-349a-11ea-8935-b99defee4ddc.png)
The control sequence `ESC [ 1 T` scrolls down 1 line, and fills the new line with the background color
![image](https://user-images.githubusercontent.com/4053256/72211505-758cef80-349a-11ea-986a-b6e1dc3033e2.png)

```
clear
echo -e "\e[H\eM"
```
![image](https://user-images.githubusercontent.com/4053256/72211539-124f8d00-349b-11ea-9733-042c0eea65fb.png)
The control sequence `ESC M` also scrolls down 1 line, but uses color 0 instead of the background color
![image](https://user-images.githubusercontent.com/4053256/72211542-23000300-349b-11ea-99d2-2bd037fddd01.png)

Applications like `less`, `man`, and `vim` all use `ESC M` for scrolling, and scrolling upwards causes new lines to be filled with the wrong background color:
![image](https://user-images.githubusercontent.com/4053256/72211655-01078000-349d-11ea-8242-08888480eae7.png)

`ESC [ L` (insert lines) appears to have the same problem.

Additionally, this only happens when the current background color is set to the default. otherwise, ESC M acts correctly (try inserting `\e[41m` after the `H` in the command)

# Expected behavior

Both control sequences should have the same behavior

# Actual behavior

`ESC M` uses color 0 instead of the default color, when the background color is set to the default

**Comment by j4james** [comment]
I believe this was fixed by PR #2987.

**Comment by zadjii-msft** [comment]
This sure does look like that bug to me as well. Thanks for finding the dupe @j4james!

/dup #2702
